src/components/Introduction.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Wrapper, MainText } from './GlobalStyles';
+
+export const Introduction = () => {
+  return (
+    <IntroductionWrapper primary>
+      <MainText primary>Hello! My name is Kartika,
+        and I&apos;m a curious front-end developer from Indonesia
+        who&apos;s currently based in Sweden.
+        I am an eager learner and always committed to finish all tasks at hand.
+      </MainText>
```

Comment:
Why make them two seperate element? Maybe add some padding/margin if you want them to be seperated in the browser?